Validation/HGCalValidation/plugins/HGCalDigiValidation.cc

```diff
@@ -336,9 +354,17 @@ void HGCalDigiValidation::fillOccupancyMap(std::map<int, int>& OccupancyMap,
 
 void HGCalDigiValidation::fillDigiInfo(digiInfo& hinfo) {
   int ilayer = hinfo.layer;
-  charge_.at(ilayer)->Fill(hinfo.charge);
-  DigiOccupancy_XY_.at(ilayer)->Fill(hinfo.x, hinfo.y);
-  ADC_.at(ilayer)->Fill(hinfo.adc);
+  //charge_.at(ilayer)->Fill(hinfo.charge);
```

Comment:
why is this histogram removed?

Reply:
The original histogram is filling the TOA information. So I just renamed the histogram from "charge_" to "TOA_".

Reply:
Okay, then please delete all commented-out lines of code in the file.

Reply:
Done removing commented-out lines.